cmd/entrypoint/main.go

```diff
@@ -36,15 +37,30 @@ var (
 func main() {
 	flag.Parse()
 
-	entrypoint.Entrypointer{
+	e := entrypoint.Entrypointer{
 		Entrypoint: *ep,
 		WaitFile:   *waitFile,
 		PostFile:   *postFile,
 		Args:       flag.Args(),
 		Waiter:     &RealWaiter{},
 		Runner:     &RealRunner{},
 		PostWriter: &RealPostWriter{},
-	}.Go()
+	}
+	if err := e.Go(); err != nil {
```

Comment:
does this block of code make sense inside the `Go` func?

Reply:
I initially thought of puting it there — but it meant it was harder to "test" the `Go` function as it would call `os.Exit`.

Reply:
fair enough :)